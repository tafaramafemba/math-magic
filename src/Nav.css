* {
  font-family: "Comic Neue", cursive;
  font-weight: 700;
}

nav {
  display: flex;
  justify-content: space-between;
  margin-left: 5px;
  margin-right: 5px;
}

.ul {
  display: inline-flex;
  flex-direction: row;
  gap: 10px;
  text-decoration: none;
  list-style: none;
  align-items: center;
}

.link {
  border-left: 1px solid #000;
  padding-left: 5px;
}

.h3 {
  font-size: 30px;
}
