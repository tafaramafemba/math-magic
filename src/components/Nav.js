import '../Nav.css';
import { Link } from 'react-router-dom';

function Nav() {
    return (
      <div>
        <nav>
          <h3 className='h3'>Math Magicians</h3>
          <ul className='ul'>
            <Link to="/">
            <li>Home</li>
            </Link>
            <Link to="/calculator">
            <li className='link'>Calculator</li>
            </Link>
            <Link to="/quote">
            <li className='link'>Quote</li>
            </Link>
            
          </ul>
        </nav>
      </div>
    );
  }

export default Nav;