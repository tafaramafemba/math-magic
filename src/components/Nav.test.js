import renderer from 'react-test-renderer'
import Nav from "./Nav";
import { BrowserRouter}  from 'react-router-dom';

describe(' Nav Component Rendering ', () =>{
    it('renders correctly', () => {
        const tree = renderer
            .create( <BrowserRouter><Nav /></BrowserRouter>)
        expect(tree).toMatchSnapshot();
    });
})
