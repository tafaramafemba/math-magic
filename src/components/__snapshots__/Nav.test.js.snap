// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` Nav Component Rendering  renders correctly 1`] = `
<div>
  <nav>
    <h3
      className="h3"
    >
      Math Magicians
    </h3>
    <ul
      className="ul"
    >
      <a
        href="/"
        onClick={[Function]}
      >
        <li>
          Home
        </li>
      </a>
      <a
        href="/calculator"
        onClick={[Function]}
      >
        <li
          className="link"
        >
          Calculator
        </li>
      </a>
      <a
        href="/quote"
        onClick={[Function]}
      >
        <li
          className="link"
        >
          Quote
        </li>
      </a>
    </ul>
  </nav>
</div>
`;
